guides/de-fr-xinvoice.mdx

```diff
@@ -0,0 +1,611 @@
+---
+title: "Germany & France - X-Invoice"
+description: Issue invoices in Germany (X-Rechnung & ZUGFeRD) and France (Factur-X)
+---
+
+## Introduction
+
+X-Invoice covers the German and French electronic invoice standards. The integration is the same for both countries. The supported standards are:
+
+- **X-Rechnung**: one of the two standards used in Germany. It consists on an XML file. It can be implemented using two different *syntaxes*, CII and UBL. They both contain the same information, but include different field names.
+- **ZUGFeRD**: the other standard used in Germany. It a hybrid format that consists on a PDF document with an embedded X-Rechnung file. The only supported syntax for this case is CII.
+- **Factur-X**: the standard used in France. It consists on a PDF document with an embedded XML file. It is worth noting that ZUGFeRD and Factur-X are a joint project between Germany and France, and their content is identical.
+
+<Tip>
+    When referring to syntaxes, it is important to note that X-Rechnung, ZUGFeRD and Factur-X are all *semantic data models*. What this means is that they define the data that must be present in the invoice, but not the actual format of the data or the explicit name of the fields.
+    
+    To implement the semantic models, there are two syntaxes approved by the EU: UN/CEFACT Cross Industry Invoice ([CII](https://unece.org/trade/documents/2023/10/executive-guide-einvoicing-cross-industry-invoice)) and OpenOasis Universal Business Language ([UBL](https://docs.oasis-open.org/ubl/UBL-2.1.html)).
+</Tip>
+
+## Prerequisites
+
+To convert invoices to X-Invoice format, ensure you have:
+
+- A valid API key and a basic understanding of the [workflow engine](guides/workflows) and [API](api-ref/introduction).
+- An invoice with:
+  - Supplier details (company name and required identities). Check the [German](https://docs.gobl.org/regimes/de) and [French](https://docs.gobl.org/regimes/fr) regimes for specifics.
+  - Line items with name, price, and applicable taxes.
+
+## Setup
+
+To issue invoices, configure a workflow in the [Invopop Console](https://console.invopop.com) with the following steps:
+
+- **Sign Envelope**: The envelope must be signed before processing. This can also be done in the **Add Sequential Code** step, which handles the invoice number.
+- **Convert to X-Invoice**: Generates the X-Invoice XML document. Configure the step to one of the following formats:
+  - **XRechnung CII**: German X-Rechnung (v3.0.2) in CII syntax (XML only).
+  - **XRechnung UBL**: German X-Rechnung (v3.0.2) in UBL syntax (XML only).
+  - **ZUGFeRD**: German ZUGFeRD (v2.3.2) in hybrid format (PDF with embedded XML).
+  - **Factur-X**: French Factur-X (v1.07.2) in hybrid format (PDF with embedded XML).
```

Comment:
I'd just use the higher level versions so we don't have to constantly update the docs [e.g. X-Rechnung (v3) instead of X-Rechnung (v3.0.2)]